services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/__tests__/__snapshots__/current_subscription.test.jsx.snap

```diff
@@ -87,7 +87,23 @@ exports[`CurrentSubscription container when there is a current subscription shou
             >
               <div>
                 <_default
-                  content="School Paid"
+                  content={
+                    <span>
+                      School Premium
+                      <Tooltip
+                        tooltipText="You have a School Paid subscription"
```

Comment:
Don't we want this to say School Premium now? That makes me feel like something is going awry.